### PR TITLE
Adding a revert policy

### PR DESCRIPTION
**Pull request by RichardLitt**
This comes directly from ipfs/go-ipfs#3166, and @Kubuxu. I have added it here and edited it slightly for language.

### CODE REVIEW

File: contribution-guidelines.md [hunk 1]
```diff
@@ -211,6 +211,28 @@ Before the pull request is merged, make sure that you squash your commits into l
 
 We use LGTM (Looks Good To Me) in comments on the code review to indicate acceptance. A change **requires** LGTMs from the maintainers of each component affected. If you know whom it may be, ping them. If not, ping [@jbenet](https://github.com/jbenet).
 
+### Reverting Changes
+
+When some change is introduced, and we decide that it isn't beneficial and/or it causes problems, we need to revert it.
+
+To make the review process and the changes easier, use git's `revert` command to revert some changes.
```
Comment (Kubuxu) [inline contribution-guidelines.md]: `revert those changes`

File: contribution-guidelines.md [hunk 2]
```diff
@@ -211,6 +211,28 @@ Before the pull request is merged, make sure that you squash your commits into l
...
+
+This suits a few purposes. First, it is much easier to see if some change was reverted by just looking into the history of the file. Imagine a commit with the title: _Add feature C_. There are many ways one could form the title for a commit reverting it, but by using `git revert`, it will be _Revert: "Add feature C"_ and thus very clear.
+Second, by using `git revert` we are sure that all changes were reverted. It is much easier to star again for state 0 and apply changes on it, than try to see if some transformation transforms state 1 to state 0.
```
Comment (Kubuxu) [inline contribution-guidelines.md]: `s/to star/to start`

**Comment by RichardLitt** [comment]
@Kubuxu Approve changes?

**Comment by Kubuxu** [comment]
Yup LGTM.

**Pull request merged**